earth_enterprise/src/common/khxml/khxml.cpp

```diff
@@ -375,7 +362,7 @@ WriteDocumentImpl(DOMDocument *doc, const std::string &filename) throw()
       } else {
         ToXMLStr fname(filename);
         LocalFileFormatTarget formatTarget(fname);
-        DOMLSOutput* lsOutput = impl->createLSOutput();
+        DOMLSOutput* lsOutput = impl->createLSOutput(&memoryManager);
```

Comment:
I'm not sure and the Xerces documentation isn't great. Here's the documentation on setByteStream: https://xerces.apache.org/xerces-c/apiDocs-3/classDOMLSOutput.html#ab11c2930780418bc2a0a5a61c3da834d. There's no clear indication of whether it could throw or not. I suppose I could wrap it in a try/catch just to be safe.

FWIW, this code hasn't really changed, so apparently it works reasonably well. That doesn't mean we shouldn't fix it, though.